rca/utils/models.py

```diff
@@ -664,3 +666,66 @@ class Sector(SluggedTaxonomy):
     class Meta:
         verbose_name = "Innovation RCA sector"
         verbose_name_plural = "Innovation RCA sectors"
+
+
+class ContactFieldsMixin(models.Model):
```

Comment:
All the fields on this model can be blank, should at least one (maybe title) be required?

Reply:
It's a good point, we handle a default title value field in the template https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/project_styleguide/templates/patterns/organisms/contact/contact.html#L9

The snag here is that some pages _do_ have a contact title, and some don't, and contact details at the page level aren't required so I think this is the best I can get out of something re-usable like this.